blocks/library/audio/index.js

```diff
@@ -29,15 +31,51 @@ registerBlockType( 'core/audio', {
 			type: 'string',
 			source: attr( 'audio', 'src' ),
 		},
+		align: {
+			type: 'string',
+		},
 	},
 
-	edit( { attributes, setAttributes, className } ) {
-		const { src } = attributes;
+	getEditWrapperProps( attributes ) {
+		const { align, width } = attributes;
```

Comment:
I don't think we need the `width` attribute here, or in the returned object below.